Maskbook/Scene/Wallet/Balance/View/AccountCardView.swift

```diff
@@ -43,11 +44,11 @@ class AccountCardView: UIView {
     private var stackViewGradientLayer: CAGradientLayer = {
         let layer1 = CAGradientLayer()
         layer1.colors = [
-            UIColor.white.withAlphaComponent(0.08).cgColor,
```

Comment:
Conflicts with [this pull request](https://github.com/DimensionDev/Mask-iOS/pull/24)